scripts/utilities/verify-files-changed.sh

```diff
@@ -4,18 +4,14 @@ function main() {
   local changeset
   local -r base_directory="$1"
   local -r grep_pattern="$2"
-  local -r current_hash="$(git rev-parse HEAD)"
-  local -r previous_hash="$(git rev-list --tags --skip=1 --max-count=1)"
```

Comment:
Think we just need to add `--no-merges` to get back to the actual commit with files changed (not the merge commit).

See: https://git-scm.com/docs/git-rev-list#Documentation/git-rev-list.txt---no-merges

Though this won't be the tagged commit, so it will break the `git describe` bit on line 9, though this is only cosmetic and we can probably just do both commands (one to get the tagged commit, and one to get the root commit).